static/src/javascripts/projects/common/modules/experiments/ab.js

```diff
@@ -61,7 +61,7 @@ export const getSynchronousTestsToRun = memoize(
     }
 );
 
-export const getAynschronousTestsToRun = (): Promise<
+export const getAsyncTestsToRun = (): Promise<
```

Comment:
this is just fixing a typo of mine. seems like `async` is harder to mis-spell...